src/components/InputTooltip.js

```diff
@@ -109,18 +109,43 @@ class InputTooltip extends Component {
         }
     }
 
+    splitText(textToSplit, query) {
+        const escapedQuery = query.trim().replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
+        const matchedIndex = textToSplit.toLowerCase().indexOf(escapedQuery);
+        const matched = textToSplit.substr(matchedIndex, escapedQuery.length);
```

Comment:
Looks like we assume that `matchedIndex` will be always `0`. Is this right assumption? I think it's right for MediaWiki suggestions, not sure about FC suggestions
```
splitText('my example text', 'example');
// returns [' text', 'example']
```

Reply:
I did it that way at first but then removed 'before' part after seeing MW/FC results.

however, it's reasonable to assume that they may start including results from the middle of phrases so I'll add it again.